elastic/kibana
# [Fleet] Integrations UI: Adjust policies list UI

## Summary

This PR updates the "Policies" tab of the "Integration Details" page (e.g. http://localhost:5601/app/integrations#/detail/linux-0.3.10/policies) with a few new pieces of functionality: 

- Adds an "Actions" menu with "Add Agent", "Edit Integration", and "Delete Integration" actions
- Adds a "+" button to the "Agents" column of the table to open the "Enroll Agent" flyout if there are no enrolled agents

See designs: https://www.figma.com/proto/G3jnu6rub6x8YExOfTWoO9/Integrations-UI-%2B-Fleet?node-id=802%3A23&scaling=min-zoom

## How To Test

1. Start Kibana + ES on basic
2. Follow the in-app instructions for setting up a fleet server where agents can be registered
3. Add a new integration like Linux or Nginx via the "Integrations > Browse" flow
4. Navigate to your installed integration, then the "Policies" tab, e.g. http://localhost:5601/app/integrations#/detail/nginx-0.5.0/policies
5. The new "+" icon should appear in the Agents column of the table, and the new Actions menu should appear in the far right column

## Screenshots

<details>
<summary>Show Screenshots</summary>

![Screen Shot 2021-06-08 at 9 28 15 AM](https://user-images.githubusercontent.com/6766512/121193551-eecd7500-c83b-11eb-83eb-26861e4fe840.png)
![Screen Shot 2021-06-08 at 9 28 22 AM](https://user-images.githubusercontent.com/6766512/121193553-ef660b80-c83b-11eb-9365-c8fae196fdc8.png)
![Screen Shot 2021-06-08 at 9 28 33 AM](https://user-images.githubusercontent.com/6766512/121193554-ef660b80-c83b-11eb-92bf-03e56c2b6899.png)
</details>

### Checklist
